packages/react/src/components/Accordion/_accordion.scss

```diff
@@ -1,11 +1,12 @@
-@import '~carbon-components/scss/globals/scss/vars';
-@import '~carbon-components/scss/components/accordion/accordion';
+@use '../../globals/spacing';
+@use '../../globals/vars';
+@use '~@carbon/styles/scss/components/accordion';
```

Comment:
All `@import '~carbon-components/scss/components/accordion/accordion';` paths are now loaded from the `@carbon/styles` packages instead, and use default `_index` support in newer sass versions. Thus, accordion is imported as `@use '~@carbon/styles/scss/components/accordion';` now.